server.js

```diff
@@ -82,71 +82,86 @@ function defaultMessageGenerator(reviewers) {
      reviewers.length > 1 ? '' : ' a',
      reviewers.length > 1 ? 's' : ''
   );
-};
+}
 
-app.post('/', function(req, res) {
-  req.pipe(bl(function(err, body) {
-    var data = {};
-    try { data = JSON.parse(body.toString()); } catch (e) {}
-
-    if (data.action !== 'opened') {
-      console.log(
-        'Skipping because action is ' + data.action + '.',
-        'We only care about opened.'
-      );
-      return res.end();
+function getRepoConfig(request): Promise<Object> {
```

Comment:
Do you mean the async function prefix? The pattern I was following was to only use it where I need it. The async prefix is required in order to use await within the function. For functions like getRepoConfig which make an async request but just return a promise and don't use await, I opted not to use it. Although I am not opposed to adding it to these functions which do the request. Removing it everywhere is not an option because it is required to use await.